src/sentry/static/sentry/app/views/organizationIncidents/details/suspects.jsx

```diff
@@ -4,42 +4,136 @@ import styled from 'react-emotion';
 
 import {Panel, PanelBody, PanelItem} from 'app/components/panels';
 import {t} from 'app/locale';
+import AsyncComponent from 'app/components/asyncComponent';
+import CommitLink from 'app/components/commitLink';
+import IdBadge from 'app/components/idBadge';
+import SentryTypes from 'app/sentryTypes';
 import SideHeader from 'app/views/organizationIncidents/details/sideHeader';
+import TimeSince from 'app/components/timeSince';
+import overflowEllipsis from 'app/styles/overflowEllipsis';
 import space from 'app/styles/space';
 
-export default class Suspects extends React.Component {
-  static propTypes = {
-    // TODO: Make this a shape once we figure out data model
-    suspects: PropTypes.array,
-  };
+function Message({type, suspect}) {
+  const {message, id, repository} = suspect;
+  if (type === 'commit') {
+    return (
+      <CommitRow>
+        <MessageOverflow>
+          <span>{message.split(/\n/)[0]}</span>
+        </MessageOverflow>
+        <CommitLink commitId={id} repository={repository} />
+      </CommitRow>
+    );
+  }
+
+  return null;
+}
+
+Message.propTypes = {
+  type: PropTypes.oneOf(['commit']),
+  suspect: SentryTypes.IncidentSuspectData,
+};
+
+const Suspects = styled(
```

Comment:
nit: We usually don't wrap entire components with styled like this - is there a reason we can't just use the styled component on line 52

Reply:
I think this is what @EvanPurkhiser was suggesting in the TSC yesterday?

Reply:
I've removed the nesting (and in this case we don't even export this component so it doesn't matter too much anyway)